### ISSUE TITLE
hyperparameters of 16 frames training.

### ISSUE THREAD

**Issue by kinfeparty** [V-Sense/ACTION-Net#6]
Hello, I follow your suggestion and this is my hyperparameters of 16 frames training.

`python3 train.py --is_train --is_shift \`
 `                --dataset EgoGesture --clip_len 16 \`
`                 --shift_div 8 --wd 1e-5 --dropout 0 \`
`                 --cuda_id $cuda_id --batch_size 16 --lr_steps 10 15 20 \`
`                 --lr 25e-4 --base_model resnet50 --epochs 25`

The val accuracy is quiet lower than 8 frames result,which is 91.7 and 8 frames is 94.4.
I really want to reproduce the result of 16 frames in Egogesture. Is there any hyperparameter I need to change?

**Comment by villawang** [comment]
Hi there, I suppose it might be caused by the small batch size.
